samples/snippets/cloud-client/src/main/java/privateca/DisableCertificateAuthority.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package privateca;
+
+// [START privateca_disable_ca]
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.security.privateca.v1.CertificateAuthority.State;
+import com.google.cloud.security.privateca.v1.CertificateAuthorityName;
+import com.google.cloud.security.privateca.v1.CertificateAuthorityServiceClient;
+import com.google.cloud.security.privateca.v1.DisableCertificateAuthorityRequest;
+import com.google.longrunning.Operation;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class DisableCertificateAuthority {
+
+  public static void main(String[] args)
+      throws InterruptedException, ExecutionException, IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    // location: For a list of locations, see:
+    // https://cloud.google.com/certificate-authority-service/docs/locations
+    // caPoolName: The name of the CA pool under which the CA is present.
+    // certificateAuthorityName: The name of the CA to be disabled.
+    String project = "your-project-id";
+    String location = "ca-location";
+    String caPoolName = "ca-pool-name";
+    String certificateAuthorityName = "certificate-authority-name";
+    disableCertificateAuthority(project, location, caPoolName, certificateAuthorityName);
+  }
+
+  // Disable a Certificate Authority which is present in the given CA pool.
+  public static void disableCertificateAuthority(
+      String project, String location, String caPoolName, String certificateAuthorityName)
+      throws IOException, ExecutionException, InterruptedException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the `certificateAuthorityServiceClient.close()` method on the client to safely
```

Comment:
should this be added to the sample?

Reply:
This was suggested in the java-docs-sample repo to be included as a standard across snippets. https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#client-initialization
Should we include this to the samples to maintain standard across repos?

Reply:
nope, that looks good then - thanks!